lib/plugin.js

```diff
@@ -110,6 +110,12 @@ ManifestPlugin.prototype.apply = function(compiler) {
       }.bind(this));
     }
 
+    files = files.map(file => {
+      file.name = file.name.replace('\\', '/');
+      file.path = file.path.replace('\\', '/');
```

Comment:
`'\\'` is equivalent `/\//`
but `/\//g` expected

Reply:
```js
'sdfds\\sdfsdf\\'.replace('\\', '/') // -> "sdfds/sdfsdf\"
'sdfds\\sdfsdf\\'.replace(/\\/, '/') // -> "sdfds/sdfsdf\"
'sdfds\\sdfsdf\\'.replace(/\\/g, '/') // -> "sdfds/sdfsdf/"
```

Reply:
Good catch :+1: